.pre-commit-config.yaml

```diff
@@ -1,3 +1,11 @@
+ci:
+  autofix_prs: false
+  autoupdate_schedule: quarterly
+  submodules: false
+
+default_language_version:
+  python: python3.11
```

Comment:
Since we support python 3.10 as minimum version, should we use 3.10 also here?

Reply:
I discovered that for some reason, `pre-commit` is not able to build a venv from a specific python version, therefore I just specified `default_language_version: python:python3`

Reply:
I have never played that much with pre-commit, a question. Is the `venv` created automatically? Shouldn't it be specified as hook in the repos that are listed in the yaml file?

Reply:
Ok now I understood, I think. 

- pre-commit always create a virtual environment.
- virtualenvs can be created only with a matching version of the corresponding `python` executable, that is generally the system's python (but it could be the conda's python if the environment is enabled).
- you need to pass a generic `python3` because you cannot make assumptions about what python version is currently active.

https://pre-commit.com/#overriding-language-version
-

Reply:
Okay, got it. Thanks for the explanation!